step-automation-packages/step-automation-packages-manager/src/main/java/step/automation/packages/execution/RepositoryWithAutomationPackageSupport.java

```diff
@@ -354,24 +354,24 @@ public AutomationPackage getAutomationPackage() {
             return getInMemoryManager().getAllAutomationPackages(null).findFirst().orElse(null);
         }
 
-        public boolean isExternallyCreatedContext() {
-            return externallyCreatedContext;
+        public boolean isShared() {
+            return shared;
         }
 
         @Override
         public void close() throws IOException {
-
-            // only after isolated execution is finished we can clean up temporary created resources
-            try {
-                // remove the context from isolated automation package repository
-                log.info("Cleanup isolated execution context");
-
-                IsolatedAutomationPackageRepository.PackageExecutionContext automationPackageManager = sharedPackageExecutionContexts.get(contextId);
+            // remove the context from isolated automation package repository
```

Comment:
This comment doesn't match the code: we're only removing it for shared managers. Is this intended?

Reply:
not sure how it appeared here, rephrased it